moon-store-mac-os/moon-store-mac-os/Helpers/Extensions/Data+String.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 extension Data {
     mutating func appendStringIfNeeded(_ string: String) {
         guard let data = string.data(using: .utf8) else {
-            debugPrint("Could not convert \(string) to UTF8")
+            _ = MSErrorHelper(referenceClass: nil).handle(error: .badData)
```

Comment:
What is the reason of create a instance in this side?

Reply:
I consider we need to know when `string` convertion to `data` is not working.